asv/feed.py

```diff
@@ -43,7 +44,9 @@ class FeedEntry(object):
         Default: same as *updated*
 
     """
-    def __init__(self, title, updated, link=None, content=None, id_context=None, id_date=None):
+    def __init__(
+                    self, title, updated, link=None, content=None,
+                    id_context=None, id_date=None):
```

Comment:
I don't think this is valid PEP8 code, and even if it is, this indentation seems quite arbitrary. Just split when the line is too long.